### PR TITLE
Update library version on README

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by danielpassos** [comment]
@wei-lee @secondsun Could you give it a spin?

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Add the FH Android `aar` library in your `build.gradle` project
 
 ```
 dependencies {
-	compile 'com.feedhenry:fh-android-sdk:2.3.1'
+	compile 'com.feedhenry:fh-android-sdk:2.4.0'
```
Comment (wei-lee) [inline README.md]: does this mean we need to update the readme file as well every time there is a new release? Is there a better way to do this?

Comment (danielpassos) [inline README.md]: Not sure if it's the best way to do this, but it's what we have for now. 
We can change it for: `compile 'com.feedhenry:fh-android-sdk:+'`

Comment (wei-lee) [inline README.md]: what does "+" mean? The latest version?

Comment (danielpassos) [inline README.md]: `+` = last version
`2+` >= 2
`2.+` >= 2.0.0 < 3.0.0 (pinning the major version)

Comment (secondsun) [inline README.md]: @wei-lee In general example code and documentation must be updated for each release and regression tested.  If we want we can have templates to generate updated README's in a code quality project.  It is part of our long term goals to automate as much of this as we can.

Comment (matzew) [inline README.md]: +1

**Comment by danielpassos** [comment]
Since we are about release `2.5.0` I'm closing it

**Pull request closed without merging**